apis/pkg/v1/provider.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
```

Comment:
nitpick: since all conformance testing will live in this repo, I don't think we need to version these apis. Any changes to them will require changes in `conformance.yaml` because we are not specifying an API version there. Maybe just `apis/provider/provider.go` would be appropriate for now?